tree/10_ncf_internals/modules/promises/get_parent.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```

Comment:
could you run shellcheck on this script and ensure there are no warnings/errors?

Reply:
(the repo tests are really permissive in this regard, we should do better in new stuff)

Reply:
Ok, my bad, I used a python script but committed its ancestor written in sh :facepalm: